changes/1836.feature.rst

```diff
@@ -0,0 +1 @@
+Add support to Freebsd13 OS.
```

Comment:
We try really hard to follow style guides when referring to other operating systems in user-facing text - I'm pretty sure this should be `FreeBSD`. Also - given FreeBSD12 isn't EOL, and FreeBSD14 is on the horizon - is there any reason to narrow this to v13 specifically?

Reply:
Oh, you are right, this should be `FreeBSD`.

The main reason is that I'm tested it on v13, and I don't want to add version that I didn't test it. Also, I don't want to see, in the future, a list of 'How to install Toga in different `FreeBSD` versions', so I preferred to maintaining only last `FreeBSD` release, especially since `FreeBSD` is a rolling release OS. However, I suggest to remove the version returned by the `sys.platform` to be just `freebsd`.